packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -1005,12 +1012,26 @@ export const Table = <
                           }
                         }}
                       >
+                        <ShadowRoot>
+                          {typeof column.Header === 'string' ? (
+                            <LineClamp>
+                              <slot />
+                            </LineClamp>
+                          ) : (
+                            <slot />
+                          )}
+                          <slot name='actions' />
+                          <slot name='resizers' />
+                          <slot name='shadows' />
+                        </ShadowRoot>
```

Comment:
Is there an advantage to using a `ShadowRoot` here?

Reply:
Yes, it hides the LineClamp wrapper div and thereby avoids changing the DOM shape (which the user might be relying on, especially for table-body cells)